networking/aws_load_balancer_operator/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-multiple-ingress-through-single-alb_{context}"]
+= creating-multiple-ingress-through-single-alb
+
+The user can route the traffic to services as a part of a domain and can serve all the endpoints through a single ALB.
+
+.Prerequisites
+
+.Procedure
+
+. Create an `AWSLoadBalancerController` and three `Ingress` resources:
++
+[source,yaml]
+----
+kind: Ingress
+metadata:
+  name: example-1 <1>
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing <2>
+    alb.ingress.kubernetes.io/group.name: example <3>
+    alb.ingress.kubernetes.io/group.order: "1" <4>
+spec:
+  ingressClass: alb <5>
+  rules:
+  - host: example.com <6>
+    http:
+        paths:
+        - path: /blog <7>
+          backend:
+            service:
+              name: example-1 <8>
+              port:
+                number: 80 <9>
+  —--
+kind: Ingress
+metadata:
+  name: example-2
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/group.name: example
+    alb.ingress.kubernetes.io/group.order: "2"
+spec:
+  ingressClass: alb
+  rules:
+  - host: example.com
+    http:
+        paths:
+        - path: /store
+          backend:
+            service:
+              name: example-2
+              port:
+                number: 80
+  —--
+kind: Ingress
+  metadata:
+  name: example-3
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/group.name: example
+    alb.ingress.kubernetes.io/group.order: "3"
+spec:
+  ingressClass: alb
+  rules:
+  - host: example.com
+    http:
+        paths:
+        - path: /
+          backend:
+            service:
+              name: example-3
+              port:
+                number: 80
+----
+
+<1> Defines name of an `Ingress`.
+<2>
+<3>
+<4>
```

Comment:
This specifies the order in which the rules from the Ingresses are matched when the request is received at the load balancer.

Reply:
Addressed